api/include/opentelemetry/trace/span_context.h

```diff
@@ -110,10 +84,10 @@ class SpanContext final
   bool IsSampled() const noexcept { return trace_flags_.IsSampled(); }
 
 private:
-  const trace_api::TraceId trace_id_;
-  const trace_api::SpanId span_id_;
-  const trace_api::TraceFlags trace_flags_;
-  const bool remote_parent_ = false;
+  trace_api::TraceId trace_id_;
+  trace_api::SpanId span_id_;
+  trace_api::TraceFlags trace_flags_;
+  bool remote_parent_ = false;
```

Comment:
Removing the `const` enables us to use default (and hopefully optimized) copy constructors and assignment operators.